src/component/MakeOffer/MakeOffer.jsx

```diff
@@ -0,0 +1,25 @@
+import PrimaryButton from "../common/PrimaryButton";
+
+function MakeOffer({ nft }) {
+  return (
+    <div className="p-2 flex flex-col gap-3 w-full">
+      <div className="bg-[#3B3B3B] p-3 flex gap-2 justify-evenly rounded-2xl max-w-[135px]">
+        {nft.artistAvatar && <img src={nft.artistAvatar} />}
```

Comment:
What should render if there is no avatar? Maybe just add a span with text

Reply:
Can use the alt tag for img instead adding a span I believe they resolve the same issue.

Reply:
Alt would show up the icon of image not being rendered. So I think a span is fine with just the initials as the text